CHANGELOG.md

```diff
@@ -138,6 +138,8 @@
 - [Changed in-memory `Column.zip` to match `Table.zip`, removing the `function`
   parameter][12626]
 - [Added Regex_match for Postgres][12663]
+- [Replace `use_bankers` flag with `Rounding_Mode` in all `round`
+  methods][12641]
```

Comment:
Shall we indicate that this is a potentially breaking change?

Reply:
NetBeans has infrastructure for marking a change as **incompatible** and then rendering it as highlighted and/or in a special section: https://bits.netbeans.org/10.0/javadoc/org-openide-modules/apichanges.html